content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -126,6 +126,26 @@ For Kubernetes version {{< skew currentVersion >}}, you can use this label on Se
 
 Part of the specification used to implement [ApplySet-based pruning in kubectl](/docs/tasks/manage-kubernetes-objects/declarative-config/#alternative-kubectl-apply-f-directory-prune). This label is what makes an object an ApplySet parent object. Its value is the unique ID of the ApplySet, which is derived from the identity of the parent object itself. This ID **must** be the base64 encoding (using the URL safe encoding of RFC4648) of the hash of the group-kind-name-namespace of the object it is on, in the form: `<base64(sha256(<name>.<namespace>.<kind>.<group>))>`. There is no relation between the value of this label and object UIDs.
 
+### kubectl.kubernetes.io/restartedAt
+
+Example: `kubectl.kubernetes.io/restartedAt: "2023-03-05T15:30:00Z"`
+
+Used on: Pod
+
+This annotation indicates the time when a workload restart was triggered, using 
+the kubectl command-line tool.
+The kubelet tool updates the Pod template in a Deployment, ReplicaSet, StatefulSet
+or DaemonSet, making a change to the metadata of Pods defined by that template.
+By either setting this annotation or changing the value to a new date, the kubectl tool
+triggers a rollout and indirectly causes the launch of fresh, replacement Pods.
+
+In the above example, the annotation indicates that the Pod is part of a workload that was
+most recently restarted at 3:30 PM UTC on March 31, 2023.
```

Comment:
```suggestion
most recently restarted at 3:30 PM UTC on March 31, 2023.
```
looks like the time stamp is for 3:30pm UTC on March 5, 2023

Reply:
@reylejano please suggest me what i put here!

Reply:
Can you work out what the date string `2023-03-05T15:30:00Z` represents? Try https://dencode.com/en/date for example.

Reply:
> Can you work out what the date string `2023-03-05T15:30:00Z` represents? Try https://dencode.com/en/date for example.

Please address thecomment here and make the change